kong/tools/wrpc/proto.lua

```diff
@@ -0,0 +1,164 @@
+local pb = require "pb"
+local grpc = require "kong.tools.grpc"
+
+local grpc_new = grpc.new
+local pb_encode = pb.encode
+local setmetatable = setmetatable
+local string_format = string.format
+
+local _M = {}
+local _MT = { __index = _M, }
+
+local wrpc_proto_name = "wrpc.wrpc"
+
+local default_proto_path = { "kong/include/", "/usr/include/", }
+
+local function parse_annotation(annotation)
+  local parsed = {}
+  for kv_pair in annotation:gmatch("[^;]+=[^;]+") do
+    local key, value = kv_pair:match("^%s*(%S-)=(%S+)%s*$")
```

Comment:
This expression allows for spaces around annotations but not around the equals sign. Maybe include this?

```suggestion
    local key, value = kv_pair:match("^%s*(%S-)%s*=%s*(%S+)%s*$")
```